test/test_ipv6.cpp

```diff
@@ -4,6 +4,7 @@
 #include "srt.h"
 #include "netinet_any.h"
 
+#undef _S
 inline std::string operator "" _S(const char* src, std::size_t)
 {
     return std::string(src);
```

Comment:
Identifiers beginning with an underscore followed immediately by an uppercase letter are reserved for compiler and OS. I would suggest to remove the function and let compiler do implicit conversion from const char * to std::string. Use std::string directly if you prefer explicit conversion.